mmaction/models/tenons/backbones/resnet_i3d_slowfast.py

```diff
@@ -222,32 +223,24 @@ class ResNet_I3D_SlowFast(nn.Module):
 
     def __init__(self,
                  depth,
-                 tau=16,
```

Comment:
We have to provide sufficient comments on why we squeeze out this parameter. In the original paper, it says: "Our Fast pathway works with a small temporal stride of τ /α, where α > 1 is the frame rate ratio between the Fast and Slow pathways." Here we force τ/α to be 1 and adjust the `new_length` and `new_step` accordingly.